micrometer-core/src/test/java/io/micrometer/core/instrument/binder/logging/LogbackMetricsTest.java

```diff
@@ -62,7 +57,7 @@ void closeLogbackMetrics() {
 
     @Test
     void logbackLevelMetrics() {
-        assertThat(registry.get("logback.events").counter().count()).isEqualTo(0.0);
+        assertThat(registry.get("logback.events").functionCounter().count()).isEqualTo(0.0);
```

Comment:
Is there any difference in what gets published with a function counter vs counter, e.g. with the Prometheus registry? I don't remember off the top of my head.